src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheckTest.java

```diff
@@ -40,8 +40,9 @@ protected String getPackageLocation() {
     @Test
     public void testGetRequiredTokens() {
         final WhitespaceAroundCheck checkObj = new WhitespaceAroundCheck();
-        assertArrayEquals(CommonUtil.EMPTY_INT_ARRAY, checkObj.getRequiredTokens(),
-                "WhitespaceAroundCheck#getRequiredTokens should return empty array by default");
+        assertWithMessage(
+                "WhitespaceAroundCheck#getRequiredTokens should return empty array by default")
+                        .that(checkObj.getRequiredTokens()).isEqualTo(CommonUtil.EMPTY_INT_ARRAY);
```

Comment:
isEmpty

Reply:
Fixed. This will not be the last. I see 2 others in master.